### ISSUE TITLE
Issues with getMethodsAnnotatedWith()

### ISSUE THREAD

**Issue by Peter-Crawley** [ronmamo/reflections#327]
If you create reflections with `Reflections reflections = new Reflections("prefix here")` then you cant use `reflections.getMethodsAnnotatedWith()` as it gives a `ReflectionsException`.
```
org.reflections.ReflectionsException: Scanner MethodAnnotationsScanner was not configured
	at org.reflections.Store.get(Store.java:39) ~[reflections-0.9.12.jar:?]
	at org.reflections.Store.get(Store.java:61) ~[reflections-0.9.12.jar:?]
	at org.reflections.Store.get(Store.java:46) ~[reflections-0.9.12.jar:?]
	at org.reflections.Reflections.getMethodsAnnotatedWith(Reflections.java:478) ~[reflections-0.9.12.jar:?]
```

Currently my solution is as follows:
```java
Reflections reflections = new Reflections(new ConfigurationBuilder()
		.setUrls(ClasspathHelper.forPackage("prefix here"))
		.setScanners(new MethodAnnotationsScanner()));
```

Is this intended behavior or is this a bug?

**Comment by ljacqu** [comment]
Yes, it's intended. Use `new Reflections("prefix here", new MethodAnnotationsScanner())` for short

**Comment by Peter-Crawley** [comment]
Thanks. The documentation shown in README.md does not reflect this, maybe
update it?

**Comment by ronmamo** [comment]
thanks @Peter-Crawley , added better javadoc and readme clarifications in the next [0.10 version branch](https://github.com/ronmamo/reflections/pull/335) - you mind taking a look?
as well, if you wish to take a closer look at the readme, I'd appreciate any comments - reach out on ronmamo at gmail

**Comment by ronmamo** [comment]
also @ljacqu thanks for your continuous contributions 🙏 feel free to reach out, your comments are valuable

**Comment by ronmamo** [comment]
fixed in 0.10
